### DISCUSSION TITLE
Media Card volume always available

### DISCUSSION THREAD

**Discussion by niccolobusato** [home-assistant/frontend#5279] [Dashboard]
## The request

the new media player card is really cool but from my personal point of view is still useless because it doe not give the opportunity to change the icon and the background color when not in use.
Most of all is missing the volume slider directly on frontend (you need to click on the card and then use the controls inside the pop-up)

## The alternatives

Right now i'm using Mini media player that does all those cool stuff, they also have an option to control exactly what to show and what to hide as well as stateless volume option.

## Additional information
Here a quick mockup of how your card can be implemented by my point of view. 

anyway thanks for your time and your amazing work!
<img width="332" alt="media player" src="https://user-images.githubusercontent.com/44100360/77076462-29a77b00-69f4-11ea-9e90-b78ad233e1ff.png">

**Comment by LeoCal** [comment]
Yep, I also believe having volume controls always available on the card would make a huge difference in terms of user experience!

I also take this opportunity to congratulate with @zsarnett for the amazing work done!

**Comment by zsarnett** [comment]
We should do this with just a button and covering an icon shows the volume slider

**Reply by krazos** [reply]
With apologies for reviving an old thread, have you considered porting over the volume control as it was recently implemented in the media player bar to the Media Control card (i.e., a small button to the left of the "browse media" button that toggles the volume slider)?
